sys/include/crypto/aes.h

```diff
@@ -41,11 +41,11 @@ typedef uint8_t u8;
 /* This controls loop-unrolling in aes_core.c */
 #undef FULL_UNROLL
 # define GETU32(pt) (((u32)(pt)[0] << 24) ^ ((u32)(pt)[1] << 16) ^ \
-                             ((u32)(pt)[2] <<  8) ^ ((u32)(pt)[3]))
+                     ((u32)(pt)[2] <<  8) ^ ((u32)(pt)[3]))
 # define PUTU32(ct, st) { (ct)[0] = (u8)((st) >> 24); \
-                              (ct)[1] = (u8)((st) >> 16); \
-                              (ct)[2] = (u8)((st) >>  8); \
-                              (ct)[3] = (u8)(st); }
+                          (ct)[1] = (u8)((st) >> 16); \
+                          (ct)[2] = (u8)((st) >>  8); \
+                          (ct)[3] = (u8)(st); }
```

Comment:
This is going to cause a conflict with #10347

Reply:
(Un)done 7542e03